src/dmd/backend/gflow.d

```diff
@@ -6,26 +6,50 @@
  *              Copyright (C) 2000-2018 by The D Language Foundation, All Rights Reserved
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
- * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/gflow.c, backend/gflow.c)
- * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/backend/gflow.c
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/gflow.d, backend/gflow.d)
+ * Coverage:    https://codecov.io/gh/dlang/dmd/src/master/src/dmd/backend/gflow.d
  */
 
-#if (SCPP || MARS) && !HTOD
+module dmd.backend.gflow;
 
-#include        <stdio.h>
-#include        <string.h>
-#include        <time.h>
+version (SCPP)
+    version = COMPILE;
+version (MARS)
+    version = COMPILE;
 
-#include        "cc.h"
-#include        "global.h"
-#include        "el.h"
-#include        "go.h"
-#include        "type.h"
-#include        "oper.h"
-#include        "vec.h"
+version (COMPILE)
+{
+
+import core.stdc.stdio;
+import core.stdc.stdlib;
+import core.stdc.string;
+
+import dmd.backend.cc;
+import dmd.backend.cdef;
+import dmd.backend.code_x86;
+import dmd.backend.oper;
+import dmd.backend.global;
+import dmd.backend.goh;
+import dmd.backend.el;
+import dmd.backend.outbuf;
+import dmd.backend.ty;
+import dmd.backend.type;
+
+import dmd.backend.dlist;
+import dmd.backend.dvec;
+
+void vec_setclear(size_t b, vec_t vs, vec_t vc) { vec_setbit(b, vs); vec_clearbit(b, vc); }
+
+bool Eunambig(elem* e) { return OTassign(e.Eoper) && e.EV.E1.Eoper == OPvar; }
```

Comment:
This is already defined in `gdag.d` - why not import it (or move it to a common bit)?

Reply:
Because the 'header' files are a mess at the moment, and I decided to defer dealing with that. My goal is to do the least disruptive thing to get the files converted to D. Then, start doing refactoring and removing technical debt.